### ISSUE TITLE
Fix static file serving examples

### ISSUE THREAD

**Issue by joshuaTayjo** [shuttle-hq/shuttle-docs#141]
Right now the following pattern appears in multiple areas of the documentation concerning axum:
```rust
    let router = Router::new()
        .route("/hello", get(hello_world))
        .merge(SpaRouter::new("/assets", static_folder).index_file("index.html"));
```
However SpaRouter is deprecated in axum-extras and the examples are now incorrect for modern versions of axum.

A method for serving static files which allows for SPA-style routing is to add a fallback service to the router which will always serve the index.html page from the static_folder which can then take over if SPA routing is necessary.
The example can be changed to the following:
```rust
    let router = Router::new().fallback_service(ServeDir::new(static_folder));
```
which use SeveDir from tower-http. When replaced with this the code in the examples compiles.

The issue appears in the "How Shuttle Works" section of the introduction, and the "Axum" section of the examples. Both cases could be replaced with the above method, or a more robust router could also be used.

**Comment by joshua-mo-143** [comment]
Hey, thanks for posting the issue! I meant to get around to this at some point but completely forgot about doing so. This will be amended shortly with a PR.

**Comment by ivancernja** [comment]
@joshua-mo-143 has this been amended?

**Comment by joshua-mo-143** [comment]
> @joshua-mo-143 has this been amended?

Has been amended now, it looks like I never got around to it: https://github.com/shuttle-hq/shuttle-docs/pull/155

The Rocket .mdx file still contains Axum references, I'm assuming we should get rid of those?

**Comment by joshua-mo-143** [comment]
Closed as the relevant PR was merged to fix this: #155
